docs/package.json

```diff
@@ -36,6 +36,7 @@
     "@babel/core": "^7.12.3",
     "@babel/preset-react": "^7.12.5",
     "@storybook/addon-a11y": "^6.0.28",
+    "@storybook/addon-controls": "^6.0.28",
     "@storybook/addon-docs": "^6.0.28",
     "@storybook/addon-knobs": "^6.0.28",
```

Comment:
I think it's best to remove the knobs plugin as part of this change as theses two features are meant to replace each other. Also it seems like you took on [T261652](https://phabricator.wikimedia.org/T261652) (which is a good thing, less work for me), Can you please assign it to yourself and move it to doing in accordance with our DS workflows?

Reply:
Can do, though this PR was merely intended as fixing something that I thought already worked :sweat_smile:.

I'll add a commit that drops Knobs and assign this PR.